ReplicationAcceptance/ReplicationAcceptance/ReplicationAcceptance.m

```diff
@@ -128,7 +148,28 @@ -(CDTReplicator *) pullFromRemoteWithFilter:(NSString*)filterName params:(NSDict
         [NSThread sleepForTimeInterval:1.0f];
         NSLog(@" -> %@", [CDTReplicator stringForReplicatorState:replicator.state]);
     }
-    
+
+    XCTAssertGreaterThanOrEqual(interceptor.timesRequestIntercepted, 1);
+    int timesRequestIntercepted = interceptor.timesRequestIntercepted;
+
+    [pull clearInterceptors];
+
+    replicator = [self.replicatorFactory oneWay:pull error:&error];
+    XCTAssertNil(error, @"%@", error);
+    XCTAssertNotNil(replicator, @"CDTReplicator is nil");
+
+    NSLog(@"Replicating from %@", [pull.source absoluteString]);
+    if (![replicator startWithError:&error]) {
+        XCTFail(@"CDTReplicator -startWithError: %@", error);
+    }
+
+    while (replicator.isActive) {
+        [NSThread sleepForTimeInterval:1.0f];
+        NSLog(@" -> %@", [CDTReplicator stringForReplicatorState:replicator.state]);
+    }
+
+    XCTAssertEqual(timesRequestIntercepted, interceptor.timesRequestIntercepted);
+
```

Comment:
This is an inappropriate addition of functionality to this method as I don't think adding overhead to every test is wise. In addition, it doesn't appear to be testing anything which isn't tested in your unit tests?